pkgs/desktops/gnome-3/apps/file-roller/default.nix

```diff
@@ -24,6 +24,11 @@ stdenv.mkDerivation rec {
     patchShebangs data/set-mime-type-entry.py
   '';
 
+  postFixup = ''
+    wrapProgram "$out/bin/file-roller" \
```

Comment:
Please add a comment that this is workaround for https://gitlab.gnome.org/GNOME/file-roller/-/issues/40.

Reply:
done, thanks for review

Reply:
Looks like cpio is not necessary here if we use the patch I accidentally pushed https://gitlab.gnome.org/GNOME/file-roller/-/commit/af27465a3ef07622e89890ef4b1aacbb754b6fb9

Reply:
Patch added. Thanks